templates/partials/filters/report-type.html

```diff
@@ -1,211 +1,211 @@
 <div class="filter">
```

Comment:
I know, it really should be. Not sure who originally set it up like this, but if someone could just build an array of the report types that would be great. Really all of the filter partials with long lists should be refactored this way.